### PR TITLE
File-upload initial commit.

### PR DESCRIPTION
**Pull request by alekhrycaiko**

### CODE REVIEW

File: src/components/DropArea.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    files: acceptedFile
+    });
+  }
+  //    onOpenClick () {
+  //      this.dropzone.open();
+  //    }
+  render() {
+    return (
+      <div>
+          <Dropzone ref={(node) => { this.dropzone = node; }} onDrop={this.onDrop}>
```
Comment (Siunami) [inline src/components/DropArea.js]: ref={(node) => { this.dropzone = node; }}

I'm not sure what this code is for

File: src/components/DropArea.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+              <div>Try dropping some files here, or click to select files to upload.</div>
+          <button type="button" onClick={this.onOpenClick}>
```
Comment (Siunami) [inline src/components/DropArea.js]: onClick is calling a function that you have commented out?

File: src/components/DropArea.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+              Open Dropzone
+          </button>
+          {this.state && this.state.files && this.state.files.length > 0 ? <div>
```
Comment (Siunami) [inline src/components/DropArea.js]: Can you add some documentation to explain what 26-30 is doing?

Comment (alekhrycaiko) [reply src/components/DropArea.js]: I think we should avoid documenting and commenting if possible. I can re-factor the format to make it more clear though, I agree that the formatting is a bit funny. This is mostly taken from the example app provided by the drop zone plugin.

**Pull request merged**